src/stravalib/tests/unit/test_model.py

```diff
@@ -322,7 +321,7 @@ def test_subscription_update_deser(self):
             datetime(2024, 4, 28, 12, 0),
             None,
         ),
-        ("Foo", None, ParserError),
+        ("Foo", None, ValueError),
```

Comment:
Since `naive_datetime()` is a Pydantic validator, it should raise `ValueError`s.